io.asgardeo.java.oidc.sdk/src/test/java/io/asgardeo/java/oidc/sdk/validators/IDTokenValidatorTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Since we updating the license header, lets check if we can update LLC as well.

Reply:
Addressed